lists/cl/cl-mp.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": {
+    "en": "Public Market Suppliers and Buyer Units, Chile",
+    "local": "Proveedor y Unidad Comprador Mercado Público, Chile"
+  },
+  "url": "https://www.mercadopublico.cl/Home",
+  "description": {
+    "en": "Mercado Público is an electronic platform managed by Chilecompra, part of the Ministry of Finance, which public organizations use to make purchases.\n\nOrganizations registered as buyers or suppliers on the Mercado Público platform are assigned an identifier in the system. This identifier is used in the data from the Mercado Público OCDS API.\n\nThe use of Mercado Público identifiers is considered a temporary measure. They should only be used when no other stable identifier for a government agency is available"
```

Comment:
I think the last sentence should read "They should only be used when no other stable identifier for a government agency or company is available"

Reply:
Good spot! Fixed in https://github.com/org-id/register/pull/424/commits/ca9c0f0eec1e827ccb53925860ca46ecf386ac58